cmd/golden/withWorkspace.yaml

```diff
@@ -0,0 +1,12 @@
+automerge: false
+parallel_apply: true
+parallel_plan: true
+projects:
+- autoplan:
+    enabled: false
+    when_modified:
+    - '*.hcl'
+    - '*.tf*'
+  dir: .
+  workspace: .
```

Comment:
Have you tested this on a real atlantis repo? I'm not sure the "." would work as a workspace name